src/__configuration__/themes/default.ts

```diff
@@ -0,0 +1,35 @@
+import * as PXBThemes from '@pxblue/react-themes';
+import * as Colors from '@pxblue/colors';
+import circles from '../../app/assets/themes/circles.svg';
+import { SiteConfig } from './types';
+
+export const defaultTheme: SiteConfig = {
+    theme: {
+        ...PXBThemes.blue,
+        overrides: {
+            ...PXBThemes.blue.overrides,
+            MuiAppBar: {
+                ...PXBThemes.blue.overrides?.MuiAppBar,
+                colorDefault: {
+                    color: Colors.white[50],
+                    backgroundColor: Colors.blue[500],
+                },
+                colorPrimary: {
+                    color: Colors.white[50],
+                    backgroundColor: Colors.blue[500],
+                },
+                colorSecondary: {
+                    color: Colors.white[50],
+                    backgroundColor: Colors.blue[700],
+                    '& .MuiInputBase-root': {
+                        color: Colors.white[50],
+                    },
+                    '& .MuiSelect-icon': {
+                        color: Colors.white[50],
+                    },
+                },
+            },
```

Comment:
Why are we overriding our default (PX Blue) theme for our default theme?

Reply:
Because it will otherwise be a black500 on blue500 for resource page & roadmap page

Reply:
So, should we be making this change in the themes repo then?

Reply:
That is correct. 

I was thinking that this will happen when we do all the dark theme updates.

Reply:
Can you maybe add a comment to this effect in whatever story we have on the backlog for dark theme updates?